include/aspect/postprocess/visualization/surface_strain_rate_tensor.h

```diff
@@ -0,0 +1,69 @@
+/*
+  Copyright (C) 2015 - 2021 by the authors of the ASPECT code.
+
+  This file is part of ASPECT.
+
+  ASPECT is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2, or (at your option)
+  any later version.
+
+  ASPECT is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+
+  You should have received a copy of the GNU General Public License
+  along with ASPECT; see the file LICENSE.  If not see
+  <http://www.gnu.org/licenses/>.
+*/
+
+
+#ifndef _aspect_postprocess_visualization_surface_strain_rate_tensor_h
+#define _aspect_postprocess_visualization_surface_strain_rate_tensor_h
+
+#include <aspect/postprocess/visualization.h>
+#include <aspect/simulator_access.h>
+
+#include <deal.II/numerics/data_postprocessor.h>
+
+
+namespace aspect
+{
+  namespace Postprocess
+  {
+    namespace VisualizationPostprocessors
+    {
+      /**
+       * A class derived from DataPostprocessor that takes an output vector
+       * and computes a variable that represents the 4 or 9
+       * components (in 2d and 3d, respectively) of the strain rate tensor at every
+       * point of the surface of the domain.
+       * The strain rate is defined as $\varepsilon(\mathbf u)
+       * - \tfrac 13 \textrm{trace}\ \varepsilon(\mathbf u) \mathbf 1 =
+       * \varepsilon(\mathbf u) - \frac 13 (\nabla \cdot \mathbf u)
+       * \mathbf I$.  The second term is zero if the
```

Comment:
This comment is not quite correct (but I know that you copied it from the `strain_rate_tensor.h` file). The correct description is in the `.cc` file in the doc string at the bottom. 

Would you mind copying the description from the .cc file here (and while you're there, also making the same change in `strain_rate_tensor.h`)?